open-banking-brasil-financial-api-1_ID2-ptbr.md

```diff
@@ -272,10 +272,13 @@ Nome: cnpj, Tipo: Array of Strings, Array Element Regex: '^\d{14}$'
 * **LoA2**: mecanismo de autenticação com a adoção de um único fator
 * **LoA3**: mecanismo de autenticação com múltiplos fatores de autenticação
 
-A seguinte regra deve ser adotada para o mecanismo de autenticação:
+A seguinte orientação deve ser observada para o mecanismo de autenticação:
+* De acordo com o Art. 17 da Resolução Conjunta nº 01, as instituições devem adotar  procedimentos e controles para autenticação de cliente **compatíveis com os aplicáveis ao acesso a seus canais de atendimento eletrônicos**.
+* Em observância à regulação em vigor, sugere-se que:
+  * **Para a autenticação do usuário em autorizações de acessos às APIs de compartilhamento de dados (Fase 2)**, os _Authorization Servers_ **deveriam** adotar, no mínimo, método compatível com `LoA2`; e
+  * **Para a autenticação do usuário em autorizações de acessos às API´s das fases subsequentes**, os _Authorization Servers_ **deveriam** adotar método de autenticação compatível com `LoA3` ou superior.
 
-* **Para controle de acesso às API´s definidas na FASE 2 (leitura de dados)**: os `Authorization Servers` das instituições transmissoras de dados devem condicionar a autenticação do usuário proprietário do dado, no mínimo, a adoção de método compatível com `LoA2`. A adoção de mecanismo de autenticação mais rigoroso (`LoA3`) fica a critério da instituição transmissora de acordo com sua avaliação de riscos.
-* **Para acesso às API´s das fases subsequentes (em especial pagamento)**: o acesso deve ser condicionado à método de autenticação compatível com `LoA3` ou superior.  
+Em todos os casos, a adoção de mecanismo de autenticação mais rigoroso (`LoA3` ou superior) fica a critério da instituição transmissora ou detentora de conta, de acordo com sua avaliação de riscos e de forma compatível com os mecanismos habitualmente utilizados. Portanto, o cliente de API **não deve** estabelecer na claim `acr` qualquer método a ser exigido, mas o método adotado pelo ASPSP **deve** ser retornado pelo _Authorization Server_ na claim `acr` conforme estabelecido nesta definição.
```

Comment:
There is a problem with the language in this section that i have seen with Santander and Banco Pan. The way acr, acr + essential vs acr_values work can not be changed with this clause. TPPs need to request one of the acr values advertised by banks on their .well-known configuration endpoints. If a Bank advertises that is supports both loa2 and loa3 but a tpp says i ONLY want loa2 (acr=loa2, essential=true) then a bank must reject this authentication attempt. There is a core part of OIDC Core and can not be overridden.

An ASPSP can choose to require loa3 but it can't technically ignore the situation when a TPP says 'i must and only want loa2'. To avoid this situation, TPPs SHOULD always request the minimum and all higher levels of assurance in their request objects that a bank is advertising.

Some examples for information:
Bank - Supports Loa2 and Loa3. 
TPP Requests loa2

Bank can reply with loa2 or l03.

Bank - Support loa2 and Loa3
TPP Request loa3

Bank can reply with loa2 or l03.

Bank - Support loa2 and Loa3
TPP Requests loa2 + essential 

Bank decides that loa2 is good enough = OK.
Bank decides that loa3 is required = FAIL / Reject.

Bank - Support loa2 and Loa3
TPP Requests loa3 + essential 
Bank MUST authenticate to loa3

Reply:
ok, so (on request) acr value could be empty (omitted) and essential set to true... this way ASPSP will choose the method and have to return it in id_token, right? This is the behavior we want to archive...

Reply:
Exactly - i put that in the next recommendation. if you put this as a MUST (or a SHOULD) under the client provision section then we can write tests for it.